### PR TITLE
libvirt, libvirt-glib and virt-viewer updated

### PR DESCRIPTION
**Pull request by KrullKorg**

### CODE REVIEW

File: mingw-w64-libvirt/PKGBUILD [hunk 1]
```diff
@@ -19,9 +19,8 @@ depends=("${MINGW_PACKAGE_PREFIX}-gnutls"
          "${MINGW_PACKAGE_PREFIX}-portablexdr")
 makedepends=("${MINGW_PACKAGE_PREFIX}-gcc" "${MINGW_PACKAGE_PREFIX}-pkg-config")
 options=('strip' 'staticlibs')
-source=(https://libvirt.org/sources/${_realname}-${pkgver}.tar.xz{,.asc})
-sha256sums=('e430e69b27d3f6c97255e638617b59b179618d531d81ac7dfe0783a1c1eeafd1'
-            'SKIP')
+source=(https://libvirt.org/sources/${_realname}-${pkgver}.tar.xz)
```
Comment (Chocobo1) [inline mingw-w64-libvirt/PKGBUILD]: I don't think you should remove `{,.asc}`

Comment (KrullKorg) [inline mingw-w64-libvirt/PKGBUILD]: i agree, but otherwise it didn't build

Comment (Chocobo1) [inline mingw-w64-libvirt/PKGBUILD]: `makepkg --skippgpcheck`
that's a lame excuse for removing it.

Comment (KrullKorg) [inline mingw-w64-libvirt/PKGBUILD]: and what's the difference? in both cases the security is bypassed

Comment (Chocobo1) [inline mingw-w64-libvirt/PKGBUILD]: And I just told you a way to bypass it for convenience, you can choose not to...

**Pull request merged**